packages/amplify_api/example/pubspec.yaml

```diff
@@ -30,6 +30,9 @@ dependencies:
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^0.1.3
 
+  # TODO: Datastore dependencies temporarily added in API. Eventually they should be moved to core or otherwise reconciled to avoid duplication.
```

Comment:
question: Does this need to be in example app if you are adding it to api plugin interface? See below comments.

Reply:
Yes, see comment above.